invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/collections/collection.html

```diff
@@ -0,0 +1,31 @@
+{# -*- coding: utf-8 -*-
+
+This file is part of Invenio.
+Copyright (C) 2024 CERN.
+
+Invenio is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+
+{%- block javascript %}
+{{ super() }}
+{{ webpack['invenio-app-rdm-collections-search.js'] }}
+{%- endblock %}
+
+{%- block page_body %}
+{{ super() }}
+<div class="ui container fluid theme-tertiary page-subheader rel p-10 mr-0-mobile ml-0-mobile">
+  {%- if collection.logo %}
+  <img class="ui rounded image logo" src="{{ collection.logo }}" alt="" />
```

Comment:
There is no `logo` in the model, how is this getting resolved?

Reply:
For jinja this resolves to a `None`. The problem would be `collection.logo.somethingelse`.

Nonetheless, it's a leftover so thanks for the heads up!